lib/tasks/book_image_url.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+desc "Check Book Image URLs"
+task :book_image_url do
+  require "net/http"
+  File.open(File.join(Rails.root, "app", "views", "doc", "_ext_books.erb")).each { |line|
+    # Check every img element to ensuer the src attribute is valid
+    if ((match = line.match(/.*<img\ssrc="([http|https].*)">/)))
```

Comment:
We use nokogiri elsewhere. We could probably use it here to get more robust parsing. Or maybe not, as I guess these templates are not fully-formed HTML. I'm not sure how an HTML parser would fare.

Reply:
I initially wanted to use `nokogiri` so that I could get more context of the missing image to provide better feedback than simple the bad URL (say grab the book name from the parent `li`). But for some reason I couldn't get rake to include the gem properly. I'll look at this, as it's definitely more ideal.

Reply:
@smithmf thanks for the effort so far! have you been able to look at this? do you need any help?